assets/product-model.js

```diff
@@ -54,5 +54,8 @@ window.ProductModel = {
 };
 
 window.addEventListener('DOMContentLoaded', () => { 
+  if (Shopify.designMode) {
+    document.querySelectorAll("[data-shopify-xr-hidden]").forEach(element => element.classList.add('hidden'));
```

Comment:
For this Editor Mobile view, I feel that second button could be **displayed** inside the editor instead of adding `display: none` to it. This would allow the merchant to see and tweak its colors. 

The weird part about it, though, is that merchants wouldn't be able to launch the actual XR (so the button would act as disabled). AFAIK, the `shopify-xr` script does not work in Editor mode.

I’m also okay with this current solution as it matches the decision we previously made (to have a whitespace).

Reply:
But then not all mobile device support the `view in your space` thing right ? So we also need that mentioned.

Reply:
@LucasLacerdaUX is bringing a good point.

We'll need to keep monitoring merchant's experience when they want to validate how the feature will work/display for them and their customers when editing their theme. If they don't see any button in the editor view, regardless if it is compatible or not, they might think it is a bug and could contact support advisor, etc. I am surprised we don't have any insights on that experience yet, isn't it something that was happening on Debut and Narrative already?

On the other side, we don't want to show an invalid layout to merchants as they would again perceive it as a bug because they wouldn't be able to compare what is the reality for the same reasons.

A potential missed opportunity is to have a fallback CTA label/tooltip in place that informs buyers there is a 3D model available and that they should consider viewing this product on the proper device to have the right experience. cc. @bldelgado Do you recall if this was explored? 🤔

For the short term, I don't mind having this fix until we find the right experience, but in the meantime, I would hope that merchants help docs could inform merchants that they could never see the `View in your space` button in the editor mode. Thoughts? _(I am not sure where in the help docs this information could live, at the [product media type](https://help.shopify.com/en/manual/products/product-media/product-media-types#3d-models) level but more at the theme customization detail level?)_

> This would allow the merchant to see and **tweak its colors**

I don't think this is the biggest concern as the theme color scheme doesn't allow you to tweak granular colors like that.

Reply:
> But then not all mobile device support the view in your space thing right ? So we also need that mentioned.
Unless it shows on any device and takes you to the app store to use a compatible app to view it 

Most Android/Apple devices will support it. It does not require an app and uses the native functionality of  `iOS Quicklook` or `Android Scene Viewer` if the device is new enough.